includes/abstracts/llms.abstract.notification.view.php

```diff
@@ -542,17 +549,24 @@ public function get_icon_src() {
 	}
 
 	/**
-	 * Get available merge codes for the current notification
+	 * Get available merge codes for the current notification.
 	 *
 	 * @since 3.8.0
 	 * @since 3.11.0 Unknown.
+	 * @since [version] Cache merge codes.
 	 *
 	 * @return array
 	 */
 	public function get_merge_codes() {
-		$codes = array_merge( $this->get_merge_code_defaults(), $this->set_merge_codes() );
-		asort( $codes );
-		return apply_filters( $this->get_filter( 'get_merge_codes' ), $codes, $this );
+
+		if ( ! isset( $this->merge_codes ) ) {
```

Comment:
```suggestion
		if ( empty( $this->merge_codes ) ) {
```
This change will allow extended classes to empty the cache without them having to call `get_merge_codes()` to refill it.

Reply:
I don't want this, I want the `$this->merge_codes` to be even empty, if it's empty, but set, the code in this if block must not be executed every time...